test/e2e/multihoming.go

```diff
@@ -736,6 +736,42 @@ var _ = Describe("Multi Homing", func() {
 					By("asserting the *client* pod can contact the underlay service")
 					Expect(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).To(Succeed())
 				})
+
+				Context("with multi network policy blocking the traffic", func() {
+					It("can not communicate over a localnet secondary network from pod to the underlay service", func() {
+						labels := map[string]string{"name": "access-control"}
+						clientPodConfig := podConfiguration{
+							name:        clientPodName,
+							namespace:   f.Namespace.Name,
+							attachments: []nadapi.NetworkSelectionElement{{Name: secondaryNetworkName}},
+							labels:      labels,
+						}
+						kickstartPod(cs, clientPodConfig)
+
+						By("asserting the *client* pod can contact the underlay service before creating the policy")
+						Expect(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).To(Succeed())
+
+						By("provisioning the multi-network policy")
+						policy := multiNetEgressLimitingIPBlockPolicy(
+							secondaryNetworkName,
+							metav1.LabelSelector{
+								MatchLabels: labels,
+							},
+							mnpapi.IPBlock{
+								CIDR: "1.1.1.0/24",
+							},
```

Comment:
While this policy works, maybe it makes more sense to use the CIDR of the subnet, but include the pod IP as excluded.

By doing that, we do not need to use this dummy CIDR.

Reply:
> While this policy works, maybe it makes more sense to use the CIDR of the subnet, but include the pod IP as excluded.

Done - on next push
Note that it is "egress" "to", so the exclude IP is the underlay service IP, not the pod one

Reply:
Right, good catch. That makes the test simpler then ( no need to check the pod IP).